src/Components/Article.jsx

```diff
@@ -39,6 +40,30 @@ export const Article = () => {
     }
   }
 
+  const handleIncrementVote = async () => {
+    try {
+      const comments = article.comment_count;
+      const updatedArticle = await incrementVotes(article.article_id);
+      updatedArticle.comment_count = comments;
+
+      setArticle(updatedArticle);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const handleDecrementVote = async () => {
+    try {
+      const comments = article.comment_count;
+      const updatedArticle = await decrementVotes(article.article_id);
+      updatedArticle.comment_count = comments;
+
+      setArticle(updatedArticle);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+

```

Comment:
We could add extra logic to allow a user to only vote only once in any direction and to allow them to change their mind 🙂